server/json-logger.h

```diff
@@ -65,6 +65,7 @@ class JsonLogger : vk::not_copyable {
   volatile sig_atomic_t json_traces_count{0};
 
   int64_t release_version_{0};
+  std::string_view hostname_{""};
```

Comment:
Please, use one of `std::string_view` or `vk::string_view`, but not both

Reply:
Also, no need for explicit initialisation in your case `""`. `std::string_view hostname;` is enough

Reply:
fixed :)